data/api/v1/full_spec.yaml

```diff
@@ -10803,6 +10803,20 @@ paths:
         schema:
           format: int64
           type: integer
+      - description: Include whether or not hosts are muted and when they will be unmuted
+          your hosts.
+        in: query
+        name: include_muted_hosts_data
+        required: false
+        schema:
+          type: boolean
+      - description: Include host metadata
```

Comment:
"Include additional metadata about the hosts (agent_version, processor, ...)". We should include a full list of the metadata fields that are available in the parenthesis.